### PR TITLE
Update training material (DSL2, among other things)

### PR DESCRIPTION
**Pull request by mribeirodantas**
- Update mentions to the file keyword in multiple sections
- Update fromSRA example
- Update processes section
- Fix typos
- Remove trailing whitespaces

### CODE REVIEW

File: asciidocs/modules.adoc [hunk 1]
```diff
@@ -146,21 +146,21 @@ executor >  local (6)
...
 WORLD!
-HELLO 
+HELLO
 ----
 ====
 
-NOTE: You can store each process in separate files within separate sub-folders or combined in one big file (both are valid). For your own projects, it can be helpful to see examples on public repos. e.g. Seqera RNA-Seq tutorial 
-https://github.com/seqeralabs/rnaseq-nf/tree/master/modules[here] or within nf-cores projects  https://github.com/nf-core/rnaseq/tree/master/modules/nf-core/modules[here].
+NOTE: You can store each process in separate files within separate sub-folders or combined in one big file (both are valid). For your own projects, it can be helpful to see examples on public repos. e.g. Seqera RNA-Seq tutorial
+https://github.com/seqeralabs/rnaseq-nf/tree/master/modules[here] or within nf-core projects  https://github.com/nf-core/rnaseq/tree/master/modules/nf-core/modules[here].
```
Comment (abhi18av) [inline asciidocs/modules.adoc:156]:
```suggestion
https://github.com/seqeralabs/rnaseq-nf/tree/master/modules[here] or within nf-core modules project https://github.com/nf-core/rnaseq/tree/master/modules/nf-core/modules[here].
```

Comment (abhi18av) [reply asciidocs/modules.adoc:156]: Small suggestion regarding the name of Github repo

Comment (ewels) [reply asciidocs/modules.adoc:156]: The link isn't to the modules project though, it's to the rnaseq pipeline. I think this is about file organisation right? So the link looks good but I agree that the text is a bit vague.

```suggestion
https://github.com/seqeralabs/rnaseq-nf/tree/master/modules[here] or within nf-core pipelines, such as https://github.com/nf-core/rnaseq/tree/master/modules/nf-core/modules[nf-core/rnaseq].
```

**Review by ewels — APPROVED** [review]
Thank goodness for this little check box (`Hide whitespace`):
image

LGTM 🙌🏻

File: asciidocs/modules.adoc (hunk 1 above)
Comment (abhi18av) [reply asciidocs/modules.adoc]: Ah, then the `/modules` got me - good catch Phil!

**Pull request merged**